### DISCUSSION TITLE
Only up to 4 cores/32gb? No larger machines available

### DISCUSSION THREAD

**Discussion by Findeton** [community/community#77921] [Codespaces]
### Select Topic Area

Bug

### Body

When I try to create a codespace, I can no longer select 8-core (or 16-core) machines! Has Microsoft run out of machines? I can only select 4 o 2 core machines... but yesterday 8 and 16 were available.

BTW this is a pain for us because we need more space. iI's mainly space, not cores, we have many containers running.

**Comment by Findeton** [comment]
We have opened a Ticket (2466443) with no answer yet.

**Comment by Findeton** [comment]
It looks like it's working now, now we can see the 8 and 16 cores option. We need those for the disk capacity really, not the cores. It would be great if we could configure the storage independently from the cpus.

**Reply by nick-youngblut** [reply]
@Findeton I only have the 2 & 4 cores options, but I used to have the 8 & 16 cores options. I'm not sure why I'm limited now

> if we could configure the storage independently from the cpus.

This has been an ask from the community for [a long time now](https://github.com/orgs/community/discussions/3785). It appears that GitHub will NEVER implement this, even though it would be *very* useful.
